sentry.client.config.ts

```diff
@@ -12,17 +12,22 @@ Sentry.init({
   dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: 1,
+  tracesSampleRate: ["development", "heroku"].includes(process.env.NODE_ENV)
```

Comment:
Q: Should this be `process.env.NODE_ENV` Or should it be `process.env.NEXT_PUBLIC_NODE_ENV` like below on L27?

Reply:
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser

<i>

> **Non-`NEXT_PUBLIC_` environment variables are only available in the Node.js environment, meaning they aren't accessible to the browser (the client runs in a different environment).**
>
> In order to make the value of an environment variable accessible in the browser, Next.js can "inline" a value, at build time, into the js bundle that is delivered to the client, replacing all references to `process.env.[variable]` with a hard-coded value. To tell it to do this, you just have to prefix the variable with `NEXT_PUBLIC_`.
>
> ```ini
> NEXT_PUBLIC_ANALYTICS_ID=abcdefghijk
> ```
>
> This will tell Next.js to replace all references to `process.env.NEXT_PUBLIC_ANALYTICS_ID` in the Node.js environment with the value from the environment in which you run next build, allowing you to use it anywhere in your code. It will be inlined into any JavaScript sent to the browser.

</i>

Only current instances of `.NEXT_PUBLIC_` I could find currently were all in this file:
```sh
git grep -n "\.NEXT_PUBLIC_" | cat

sentry.client.config.ts:12:  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
sentry.client.config.ts:27:    process.env.NEXT_PUBLIC_NODE_ENV
sentry.client.config.ts:36:      maskAllText: process.env.NEXT_PUBLIC_NODE_ENV === "development",
sentry.client.config.ts:37:      blockAllMedia: process.env.NEXT_PUBLIC_NODE_ENV === "development",
```
&hellip; and this looked like the only instance of an env var that WASN'T `NEXT_PUBLIC_` prefixed:
```sh
git grep -n "process\.env\." sentry.client.config.ts | grep -v "\.NEXT_PUBLIC_"

sentry.client.config.ts:15:  tracesSampleRate: ["development", "heroku"].includes(process.env.NODE_ENV)
```